js/views/drawerView.js

```diff
@@ -30,17 +38,21 @@ class DrawerView extends Backbone.View {
   }
 
   initialize() {
+    _.bindAll(this, 'onShadowClicked');
     this._isVisible = false;
-    this.disableAnimation = Adapt.config.has('_disableAnimation') ? Adapt.config.get('_disableAnimation') : false;
+    this.disableAnimation = Adapt.config.get('_disableAnimation') ?? false;
+    this.$el.toggleClass('disable-animation', Boolean(this.disableAnimation));
     this._globalDrawerPosition = Adapt.config.get('_drawer')?._position ?? 'auto';
-    this.drawerDuration = Adapt.config.get('_drawer')?._duration ?? 400;
+    const drawerDuration = Adapt.config.get('_drawer')?._duration ?? 400;
+    document.documentElement.style.setProperty('--drawerDuration', `${drawerDuration}ms`);
```

Comment:
@oliverfoster I added the `drawerDuration` as a CSS custom property. Would it make sense to do the same with Notify so that the duration is configurable via JSON? If so, where would that configuration go? There is no `_notify` property like there is with `_drawer` in _config.json_.

Reply:
Adding a notify property is fine. Are we sure we want these to be differentiated and not just relative to the @duration attribute that already exists?

Reply:
The existing `@duration` property was too short, so the Notify and Drawer were animating in too quickly. 400ms is what was previously used with velocity.

Reply:
Alternatively, we could remove the speed from the JSON config and add a new `@duration` in the [Vanilla theme](https://github.com/adaptlearning/adapt-contrib-vanilla/blob/324b1fa92f7de903a2f45e136b3317721d75f682/less/_defaults/_animations.less#L3) for larger animations. I'm not sure how often devs customize the drawer speed via JSON, but it does seem like more of a theme concern.

Reply:
I'm not sure about adding it as a json property for core, I can't imagine it will be that frequently modified.

Nor about adding it as a CSS property, as that would suggest we'd want to override it at runtime, perhaps with different durations for various types of notify?

A theme extensible property seems sensible and a larger duration of 400ms seems sensible.

So perhaps a @duration-x2 or similar? @duration-longer? Any precedence for x2 or longer?

Reply:
Yep, that sounds good. Since we have `@duration-progress` already in Vanilla (used for the PLP bar), we could create `@duration-notify` and `@duration-drawer` so that it is clear where they would be used.

Should we remove the configurable `_drawer._duration` property as well to be consistent?

Reply:
No, I can see why you did it now. Sorry.

I think add in the notify property and do them both as CSS variables - prefixed appropriately.

Yup. It was a good idea.